modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -1345,6 +1357,18 @@ definition {
 		}
 	}
 
+	macro viewFieldsPresentOnLayout {
+		Click(locator1 = "ObjectAdmin#ADD_FIELD_ON_BLOCK");
+
+		Click(locator1 = "ObjectAdmin#LAYOUT_ENTRY_CHOOSE_OPTION");
+
+		for (var fieldsTypes : list "${fieldsTypesList}") {
+			AssertElementPresent(
+				key_optionName = "${fieldsTypes}",
+				locator1 = "ObjectAdmin#LAYOUT_ADD_FIELD_OPTION");
+		}
+	}
```

Comment:
Can you make this assertion at the test level? This macro is specific for this scenario.

Reply:
Done !